lib/samples/find_route_in_mobile_map_package/find_route_in_mobile_map_package.dart

```diff
@@ -71,7 +57,7 @@ class _FindRouteInMobileMapPackageState
       body: SafeArea(
         // Display a list of maps from the mobile map packages once loaded.
         child: FutureBuilder(
-          future: mobileMapPackages,
+          future: _loadMobileMapPackages(),
```

Comment:
This causes a problem with FutureBuilder -- every time the `build()` method is called, `_loadMobileMapPackages()` will be called, which will create a new Future each time. It is important to create the Future only once. Usually it would be created `initState()`.